internal/pkg/skuba/addons/dex_test.go

```diff
@@ -0,0 +1,343 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package addons
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+	"github.com/SUSE/skuba/internal/pkg/skuba/skuba"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+func TestGetDexImage(t *testing.T) {
+	type args struct {
+		imageTag string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		// TODO: Add test cases.
+		{
+			name: "TestGetDexInitImage",
+			args: args{
+				imageTag: "2.16.0",
+			},
+			want: "registry.suse.de/devel/caasp/4.0/containers/containers/caasp/v4/caasp-dex:2.16.0",
+		},
+		{
+			name: "TestGetDexInitImage",
```

Comment:
Two test cases with the same name make it harder to check which test cases failure if happens.